src/View/BakeView.php

```diff
@@ -187,12 +187,11 @@ protected function _evaluate($viewFile, $dataForView)
     /**
      * Get the contents of the template file
      *
-     * @param string $name A template name or a Bake template string
+     * @param string $filename A template filename
      * @return string Bake template to evaluate
      */
-    protected function _getViewFileContents($name)
+    protected function _getViewFileContents($filename)
     {
-        $filename = $this->_getViewFileName($name);
```

Comment:
Thats the thing, tests ran fine with/without this change, but without this change, it seemed that we were grabbing the name twice. I haven't thought of a thoughtful reproduction yet :(